app/views/preview/sections/preview-browser-based.test.js

```diff
@@ -56,4 +56,104 @@ describe('preview-browser-based', () => {
+
+        const browserBasedSectionTable = $('[data-test-id="preview-section-table-browser-based"]');
+        const mobileResponsiveQuestionRow = browserBasedSectionTable.find('[data-test-id="preview-section-table-row-mobile-responsive"]');
+
+        expect(browserBasedSectionTable.length).toEqual(1);
+        expect(mobileResponsiveQuestionRow.length).toEqual(1);
+        expect(mobileResponsiveQuestionRow
+          .find('.nhsuk-summary-list__key').text().trim()).toEqual('Mobile responsive');
+        expect(mobileResponsiveQuestionRow
+          .find('.nhsuk-summary-list__value')
+          .find('[data-test-id="preview-question-data-text"]').length).toEqual(1);
+
+        done();
+      });
+  });
+
+  it('should render the plugins required answer', (done) => {
+    const context = {
+      section: {
+        sections: {
+          'plug-ins-or-extensions': {
+            answers: {
+              'plugins-required': 'yes',
+            },
+          },
+        },
+      },
+    };
+
+    const dummyApp = createDummyApp(context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const browserBasedSectionTable = $('[data-test-id="preview-section-table-browser-based"]');
+        const pluginsRequiredQuestionRow = browserBasedSectionTable.find('[data-test-id="preview-section-table-row-plugins-required"]');
+
+        expect(browserBasedSectionTable.length).toEqual(1);
+        expect(pluginsRequiredQuestionRow.length).toEqual(1);
+        expect(pluginsRequiredQuestionRow
+          .find('.nhsuk-summary-list__key').text().trim()).toEqual('Plug-ins or extensions required');
+        expect(pluginsRequiredQuestionRow
+          .find('.nhsuk-summary-list__value')
+          .find('[data-test-id="preview-question-data-text"]').length).toEqual(1);
+
+        done();
+      });
+  });
+
+  it('should render the plugins detail answer', (done) => {
+    const context = {
+      section: {
+        sections: {
+          'plug-ins-or-extensions': {
+            answers: {
+              'plugins-detail': 'Some plugin detail',
+            },
+          },
+        },
+      },
+    };
+
+    const dummyApp = createDummyApp(context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const browserBasedSectionTable = $('[data-test-id="preview-section-table-browser-based"]');
+        const pluginsRequiredQuestionRow = browserBasedSectionTable.find('[data-test-id="preview-section-table-row-plugins-detail"]');
+
+        expect(browserBasedSectionTable.length).toEqual(1);
+        expect(pluginsRequiredQuestionRow.length).toEqual(1);
+        expect(pluginsRequiredQuestionRow
+          .find('.nhsuk-summary-list__key').text().trim()).toEqual('Plug-ins or extensions information');
+        expect(pluginsRequiredQuestionRow.find('div[data-test-id="preview-question-data-text"]').length).toEqual(1);
```

Comment:
You don't actually check to if the answer provided in the context is rendered.

Reply:
I'll add this.